opam.export

```diff
@@ -21,6 +21,7 @@ roots: [
   "js_of_ocaml-toplevel.4.0.0"
   "lmdb.1.0"
   "menhir.20210419"
+  "memtrace_viewer.0.16.0"
```

Comment:
Was this dependency used somehow?

Reply:
only locally for memory trace. i will remove it